src/libcore/ptr.rs

```diff
@@ -946,6 +946,73 @@ pub unsafe fn write_volatile<T>(dst: *mut T, src: T) {
     intrinsics::volatile_store(dst, src);
 }
 
+/// Freezes `count * size_of::<T>()` bytes of memory, converting uninitialized data into
+/// arbitrary but fixed data.
+///
+/// Uninitialized memory has undefined contents, and interaction with those contents
```

Comment:
In https://github.com/rust-lang/rust/pull/58468, I use terminology like "uninitialized memory does not have a fixed value/content", to try and distinguish it from memory that contains unknown but fixed data. Do you think it would make sense to also use such terminology here?